doc/developer/release-checklist.md

```diff
@@ -53,6 +60,23 @@
     ./dc.sh clean-load-test
     ```
 
+  - [ ] From the VM, ensure all containers are running:
+    ```shell script
+    docker ps -a
+    ```
+
+    If the prometheus container has failed with this error:
+    ```shell script
+    level=error ts=2020-03-13T18:29:51.472Z caller=query_logger.go:82 component=activeQueryTracker msg="Error opening query log file" file=/prometheus/queries.active err="open /prometheus/queries.active: permission denied"
+    panic: Unable to create mmap-ed active query log
+    ```
+
+    Do the following:
+    ```shell script
+    sudo chmod 777 prometheus/data
+    ./dc.sh up prometheus
+    ```
```

Comment:
I think we could just add `chmod 777 prometheus/data` to dc.sh and have this work reliably for everyone?